src/main/keyboard/shortcutHandler.js

```diff
@@ -131,10 +114,6 @@ class Keybindings {
     }
   }
 
-  _fixAidou () {
```

Comment:
This and the code in the switch is still needed because we have to map the dead key to the fixed one. Please rename the method name.

Reply:
How about use `_fixDeadKey` as method name?